### DISCUSSION TITLE
Smart way to convert mermaid markdown files to images?

### DISCUSSION THREAD

**Discussion by plbremer** [squidfunk/mkdocs-material#7095] [Q&A]
I have a mermaid diagram that is unusably tiny

My wanderings have lead me to this posting ( https://github.com/squidfunk/mkdocs-material/issues/5208 ) which suggests either 
- writing my own plugin
- using a plugin that adds zoom to images

i have zero hope of doing the first in any tractable amount of time, so i was curious if people had success/strategies for transforming their mermaid diagrams to images. I tried playing with  ( https://github.com/hmajid2301/markdown-mermaid-to-images ) locally, but i got some weird errors and it looks like the package is not maintained, so I am not hopeful.

Any ideas?

**Comment by alexvoss** [comment]
Have you tried [mermaid-cli](https://github.com/mermaid-js/mermaid-cli)? I don't use it but since it is part of Mermaid itself...

**Reply by plbremer** [reply]
thanks for the suggestion @alexvoss 
For those concerned, i was able to install the javascript cli and successfully generated an image.
I was going to figure out how to manage javascript packages at the the same time (naive in that department), but it turns out that I actually want to not user mermaid at all, for reasons that are beyond the scope of this thread.

**Comment by jkinred** [comment]
A plugin which does full screen, panning and zooming Mermaid diagrams has been published. Works well for us so far:
https://github.com/PLAYG0N/mkdocs-panzoom
